content/download/releases/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
 page_template= "release-page.html"
 template= "release-section.html"
-sort_by= "date"
+sort_by= "weight"
```

Comment:
Yeah, I tested locally. Sort lexicographically by title works. With this, we don't need to add the weight field.

```suggestion
sort_by= "title"
```

Reply:
So, lexographic won't work because 8.0.2 will come after 8.0.10, but apparently sorting by title isn't actually lexographic. "Zola uses a natural sorting algorithm which means that titles like "Chapter 2" will be sorted correctly before "Chapter 10"." cool